cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -130,6 +130,9 @@ purge() {
         umount -l /run/snapd/ns/ || true
     fi
 
+    echo "Unmounting snapd-specific naming cgroup"
+    umount -l /run/snapd/cgroup || true
```

Comment:
I think you want:
```
if [ -d /run/snapd/cgroup ]; then
    umount -l /run/snapd/cgroup || true
fi
```
To avoid an error message if snapd is removed before a snap was run.